src/sass/vre.scss

```diff
@@ -6,3 +6,7 @@
 @import "modules/m-modal";
 @import "modules/m-omb-info";
 @import "modules/m-form-confirmation";
+
+.schemaform-first-field .schemaform-label, .schemaform-field-template > .usa-input-error {
```

Comment:
Do we need this? It looks like a duplicate of some styles in `_m-schemaform.scss`

Reply:
This is just an override, I'll add a comment clarifying this.

Reply:
This doesn't seem to be an issue anymore, it has been removed.